rpm/libxcrypt.spec

```diff
@@ -0,0 +1,581 @@
+# Based on Fedoras libxcrypt.spec with %{?fedora macros removed/replaced}
+
+# Build with new api?
+#%%if 0%{?fedora} >= 30 || 0%{?rhel} >= 9
```

Comment:
Taking fedora as basis is fine but there's quite a lot of things here we really don't need.

Reply:
Removed and noted in the spec file.